server/server_test.go

```diff
@@ -1488,6 +1493,71 @@ var _ = Describe("Server package test", func() {
 			testURL("GET", versionURL, "", nil, http.StatusUnauthorized)
 		})
 	})
+
+	Describe("Healthcheck", func() {
+		Context("DB and sync working", func() {
+			It("should return 200 for members", func() {
+				testURL("GET", healthcheckURL, memberTokenID, nil, http.StatusOK)
+			})
+
+			It("should return 401 for non-members", func() {
+				testURL("GET", healthcheckURL, "", nil, http.StatusUnauthorized)
```

Comment:
should authorization be required for health check?

Reply:
solved